[dynamo] Remove VariableTracker.add_options

jansel opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #112961
* #112962
* #111726
* __->__ #111725
* #111415
* #111306



cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @aakhundov @kadeng

jansel committed:
[dynamo] Remove VariableTracker.add_options

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

voznesenskym committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

voznesenskym committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

voznesenskym committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

voznesenskym committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

voznesenskym committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

voznesenskym committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

voznesenskym committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

voznesenskym committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

voznesenskym committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

voznesenskym committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

voznesenskym committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

voznesenskym committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

voznesenskym committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

voznesenskym committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

zhxchen17 commented:
@zhxchen17 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/).

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

Differential Revision: [](https://our.internmc.facebook.com/intern/diff/)

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

Differential Revision: [](https://our.internmc.facebook.com/intern/diff/)

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

Differential Revision: [](https://our.internmc.facebook.com/intern/diff/)

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

Differential Revision: [](https://our.internmc.facebook.com/intern/diff/)

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

Differential Revision: [](https://our.internmc.facebook.com/intern/diff/)

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

Differential Revision: [](https://our.internmc.facebook.com/intern/diff/)

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

Differential Revision: [](https://our.internmc.facebook.com/intern/diff/)

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

Differential Revision: [](https://our.internmc.facebook.com/intern/diff/)

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

Differential Revision: [](https://our.internmc.facebook.com/intern/diff/)

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

Differential Revision: [](https://our.internmc.facebook.com/intern/diff/)

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

Differential Revision: [](https://our.internmc.facebook.com/intern/diff/)

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

Differential Revision: [](https://our.internmc.facebook.com/intern/diff/)

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

Differential Revision: [](https://our.internmc.facebook.com/intern/diff/)

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

Differential Revision: [](https://our.internmc.facebook.com/intern/diff/)

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

Differential Revision: [](https://our.internmc.facebook.com/intern/diff/)

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

Differential Revision: [](https://our.internmc.facebook.com/intern/diff/)

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

Differential Revision: [](https://our.internmc.facebook.com/intern/diff/)

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

Differential Revision: [](https://our.internmc.facebook.com/intern/diff/)

[ghstack-poisoned]

jansel committed:
Update on "[dynamo] Remove VariableTracker.add_options"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]
